tests/Microsoft.CodeAnalysis.Testing/Directory.Build.props

```diff
@@ -2,15 +2,16 @@
 <Project>  
   <PropertyGroup>
     <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
-    
+    <NoWarn>$(NoWarn);NU5125</NoWarn>
+
     <!-- Opt-out repo features -->
     <UsingToolXliff>false</UsingToolXliff>
   </PropertyGroup>
 
-  <Import Project="Sdk.props" Sdk="RoslynTools.RepoToolset" />
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <PropertyGroup>
-    <TestTargetFrameworks>net46;netcoreapp1.1;netcoreapp2.0</TestTargetFrameworks>
+    <TestTargetFrameworks>net472;netcoreapp1.1;netcoreapp2.0</TestTargetFrameworks>
```

Comment:
:exclamation: this can't change. If you need `net472`, it can be added as a fourth framework.